Point_set_processing_3/include/CGAL/IO/read_off_points.h

```diff
@@ -190,12 +190,13 @@ bool read_OFF(std::istream& is,
           put(normal_map, pwn, normal); // normal_map[&pwn] = normal
         *output++ = pwn;
         pointsRead++;
+
       }
       // ...or skip comment line
     }
     // Skip remaining lines
   }
-
+  is.clear(is.rdstate() & ~std::ios_base::failbit); // set by getline
```

Comment:
```suggestion
  if(is.eof()) {
    is.clear(is.rdstate() & ~std::ios_base::failbit); // set by getline
  }
```

Reply:
Each time the loop `while(getline(...))` exits "normally", that is because it reached the eof, and then set the `failbit`. But the `failbit` might be for another reason: there might be another error, like an hardware I/O error.